.circleci/config.yml

```diff
@@ -8,3 +8,18 @@ workflows:
           name: lint
       - ruby-rails/test-gem:
           name: test
+          before-test:
+            - run:
+                name: Install exiftool
+                command: curl -L http://cpanmin.us | perl - --sudo Image::ExifTool
+            - run:
+                name: Install libvips
+                command: sudo apt-get update && sudo apt-get install -y libvips
+            - setup_remote_docker
+            - run:
+                name: Install Kakadu
+                command: |
+                  docker create --name kakadu josejuansanchez/kakadu:1.0
+                  docker cp kakadu:/kakadu kakadu
+                  sudo cp kakadu/*.so /usr/lib
```

Comment:
Can you add a comment about what you're doing here?

Reply:
This is what I had to do for assembly-image to pass, so I assumed it was also needed here.

Reply:
all of this is needed.  See https://github.com/IU-Libraries-Joint-Development/pumpkin/blob/master/bin/ci_kakadu_install.sh.  We also do it in assembly-image.